Add QA standards

andrewhick opened the pull request:
This is a summary of all the main standards adapted from the [QA Wiki](https://github.com/DEFRA/qa-test/wiki). If this pull request is approved then the information will be removed from the wiki.

andrewhick commented:
Thanks @ccapperEA , I've expanded a bit more on how to store passwords.

andrewhick commented:
Added reporting and compliance sections, after a nudge from Matt Wiggans to cross-check against the existing digital test strategy document.

irisfaraway commented:
I like this! Only suggestion would be that I wonder if it's big enough to split up into smaller guides, but maybe it's fine to wait and see if we want to expand on any of these topics.

andrewhick commented:
@bensagar-ea - thanks for the prompt, I've updated the contents pages and fixed a bug on the pl/sql link too.

@irisfaraway - thanks, I think you're right although I'm keen to get the first iteration live and refine based on feedback!

andrewhick commented:
We also have more detailed guides on https://github.com/DEFRA/qa-test/wiki and if we need more guidance, I'm keen to point to those instead if possible.
